data/database.py

```diff
@@ -755,6 +755,26 @@ class RobotAccountToken(BaseModel):
     fully_migrated = BooleanField(default=False)
 
 
+class QuotaLimitGroups(BaseModel):
+    group_name = CharField()
+
+
+class QuotaType(BaseModel):
+    description = CharField()
+
+
+class QuotaLimits(BaseModel):
+    quota_limit_group_id = ForeignKeyField(QuotaLimitGroups)
+    quota_type_id = ForeignKeyField(QuotaType)
+    percent_of_limit = IntegerField(default=0)
+
+
+class UserOrganizationQuota(BaseModel):
+    organization_id = QuayUserField(index=True, unique=True)
+    limit_bytes = BigIntegerField()
+    quota_limit_group_id = ForeignKeyField(QuotaLimitGroups)
+
```

Comment:
What are quota groups, and why do we need the `QuotaLimit` table? Seems like it's tracking the amount of storage in terms of percentage that an organization is using, but couldn't that be part of the same table as `UserOrganizationQuota`? Or even be computed from the `RepositorySize` table.

Reply:
I think we can remove quota groups the more I work on this and will continue to think on it for a bit. The quota limit table allows for a limit to be of type warning at 80% usage and of type error at 100% usage and provides access to other types of limits if we want to implement them at a later date.